### PR TITLE
Bug/pending primary user update

### PR DESCRIPTION
**Pull request by ross-mcfadyen**
#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes, for a new component
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: server/models/classes/user.js [hunk 1]
```diff
@@ -674,7 +674,6 @@ class User {
               null,
               this.uid,
               this.fullname,
-              err,
```
Comment (ross-mcfadyen) [inline server/models/classes/user.js:677]: Removed for eslint

File: server/models/classes/user.js [hunk 2]
```diff
@@ -795,7 +795,7 @@ class User {
     }
   }
 
-  async delete(deletedBy, externalTransaction = null, associatedEntities = false) {
```
Comment (ross-mcfadyen) [inline server/models/classes/user.js:798]: Removed for eslint as it's unused

File: src/app/features/workplace/user-account-edit-permissions/user-account-edit-permissions.component.ts [hunk 3]
```diff
@@ -15,9 +15,10 @@ import { ErrorSummaryService } from '@core/services/error-summary.service';
 import { EstablishmentService } from '@core/services/establishment.service';
 import { UserService } from '@core/services/user.service';
 import { Subscription } from 'rxjs';
-// eslint-disable-next-line max-len
+
 import { UserAccountChangePrimaryDialogComponent } from '../user-account-change-primary-dialog/user-account-change-primary-dialog.component';
 
+// eslint-disable-next-line max-len
```
Comment (aaron-russell) [inline src/app/features/workplace/user-account-edit-permissions/user-account-edit-permissions.component.ts:21]: I don't think there's a need for this comment now

**Review by aaron-russell — CHANGES_REQUESTED** [review]

File: src/app/features/workplace/user-account-edit-permissions/user-account-edit-permissions.component.ts (hunk 3 above)
Comment (ross-mcfadyen) [reply src/app/features/workplace/user-account-edit-permissions/user-account-edit-permissions.component.ts]: @aaron-russell - removed.

**Review by aaron-russell — APPROVED** [review]

**Pull request merged**